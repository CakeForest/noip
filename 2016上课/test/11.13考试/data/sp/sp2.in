100 99 100
2 1 2235
3 2 5976
4 3 1365
5 4 5714
6 5 3052
7 2 9390
8 3 866
9 8 7912
10 9 6691
11 10 4604
12 11 7407
13 12 4032
14 13 9155
15 14 9354
16 15 5055
17 16 877
18 17 1709
19 18 2258
20 19 2185
21 20 8111
22 21 1240
23 22 2670
24 23 3225
25 24 272
26 25 9812
27 26 2499
28 27 2854
29 28 5740
30 10 3115
31 30 1950
32 10 7316
33 32 8804
34 33 505
35 34 3117
36 35 1595
37 36 948
38 37 334
39 38 2945
40 11 2471
41 4 3667
42 7 8801
43 42 4386
44 43 1921
45 44 2332
46 45 7559
47 46 52
48 47 5166
49 2 11
50 49 164
51 50 6729
52 51 9079
53 52 4617
54 53 914
55 9 7208
56 55 6953
57 56 3372
58 57 7680
59 33 1628
60 59 7309
61 1 5177
62 42 1836
63 62 8816
64 63 9707
65 64 3659
66 65 9652
67 66 3017
68 67 8442
69 68 9944
70 69 2714
71 70 1604
72 71 6491
73 72 7922
74 32 5590
75 9 4690
76 75 4583
77 76 892
78 77 5536
79 78 976
80 79 2678
81 80 1453
82 81 4142
83 82 8812
84 83 2130
85 45 8442
86 85 5722
87 86 1062
88 87 1787
89 88 7679
90 89 3432
91 90 451
92 14 5420
93 92 8829
94 93 662
95 94 2024
96 95 6942
97 26 5131
98 97 5493
99 98 3653
100 99 636
4 42
34 95
87 87
82 26
17 43
63 87
32 69
12 47
9 63
1 52
55 40
32 64
45 86
83 7
100 68
87 96
53 66
4 90
75 43
84 5
43 3
34 56
57 93
7 100
30 19
62 71
62 5
39 18
44 26
85 75
4 25
89 46
21 28
2 46
94 6
74 24
94 42
25 67
90 98
53 37
46 97
73 62
89 56
83 97
96 47
3 55
61 91
17 96
71 91
30 54
78 96
89 85
57 49
45 16
34 59
65 50
24 36
4 77
95 19
90 13
4 63
95 44
49 27
62 69
75 8
89 27
12 37
75 43
60 36
98 76
11 54
74 89
9 65
18 12
64 47
66 71
89 47
80 24
37 62
34 81
47 48
12 47
47 88
66 54
40 31
89 66
1 97
98 72
85 32
51 12
5 71
1 6
3 18
78 50
52 95
20 12
40 68
97 48
49 78
40 31
