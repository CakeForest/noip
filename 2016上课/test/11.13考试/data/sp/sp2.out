25532
58758
0
103607
84506
34517
102025
67897
43597
18218
20974
67311
14164
57906
171644
112576
72461
62295
42021
51749
28553
37477
63671
117714
47566
57555
33082
67757
117909
54716
78988
32251
22572
34389
65709
74788
79745
138477
158989
64330
132931
71968
80437
117550
104961
15986
62817
39163
96605
46074
71467
16250
32298
87882
2133
42384
83219
20308
36188
92442
36184
88076
95921
53237
12602
147432
34296
42021
14154
98554
47563
85873
56963
30182
36609
25721
32303
87928
63757
44124
5166
67897
24624
73375
12140
67001
100777
188108
64033
40360
90637
18342
57662
30630
79561
34625
91840
138149
30466
12140
